src/coreclr/src/jit/lower.cpp

```diff
@@ -4498,12 +4504,43 @@ bool Lowering::TryCreateAddrMode(GenTree* addr, bool isContainable)
 // Arguments:
 //    node - the node we care about
 //
-void Lowering::LowerAdd(GenTreeOp* node)
+// Returns:
```

Comment:
This also needs to be in the (missing) function header for 
`GenTree* Lowering::LowerNode(GenTree* node)`